### ISSUE TITLE
Use without OFFLINE\Cashier\Models\User ?

### ISSUE THREAD

**Issue by StrangerPings** [OFFLINE-GmbH/oc-cashier-plugin#10]
Hi there, is it possible to use OFFLINE\Cashier\Models\User?

Reason is that when I have the plugin installed, it interferes with my user stuff like, Avatar upload and changing user details.

**Comment by tobias-kuendig** [comment]
>  it interferes with my user stuff like, Avatar upload and changing user details.

Is this something you have added yourself? The plugin should not break any of the default functionality if installed.

**Comment by andrealune** [comment]
I confirm the bug, if you upload an user avatar, for example, the _attachment_type_ set in _system_files_ table is wrong, should set the original **RainLab\User\Models\User** extended class.

**Comment by vosco88** [comment]
I can confirm as well, this is how the user avatar looks in the system files, I haven't changed anything in the defaults, just playing around for now. 
`"id"	"disk_name"	"file_name"	"file_size"	"content_type"	"title"	"description"	"field"	"attachment_id"	"attachment_type"	"is_public"	"sort_order"	"created_at"	"updated_at"
"1"	"5e07596e8589a827512399.jpg"	"img (10).jpg"	"13724"	"image/jpeg"	\N	\N	"avatar"	"1"	"RainLab\User\Models\User"	"1"	"1"	"2019-12-28 13:32:30"	"2019-12-28 13:32:32"
`

**Comment by RomainMazB** [comment]
Hi @StrangerPings @andrealune @vosco88 , I've never installed this plugin, I'm trying to integrate Cashier as a backend payment method (long story), but I saw this issue and had a work-around idea modifying the morph map of the RainLab.User model, directly in the Plugin.php. I don't have the time for now to test it for now but if someone can confirm:
```
public function boot()
{
    Relation::morphMap([
        'OFFLINE\Cashier\Models\User' => 'RainLab\User\Models\User',
    ]);
}
```
It's a weird work-around:
These lines will change the class name when submitting an avatar to set 'OFFLINE\Cashier\Models\User' when asking for 'RainLab\User\Models\User'.

But this may do the job 🤞🏼 .

**Comment by tobias-kuendig** [comment]
This problem is resolved in v2 via #18
